bundles/org.openhab.binding.modbus.helioseasycontrols/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,1065 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="modbus"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+    <thing-type id="helios-ventilation-easycontrols">
+    
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="tcp" />
+        </supported-bridge-type-refs>
+    
+        <label>Helios Ventilation</label>
+        <description>Supports controlling a Helios Heat-Recovery Ventilation Device with easyControls using the Modbus Gateway TCP</description>
```

Comment:
I'm using the name also used in the vendor's specs because I thought this should make it easier for device owners to see the binding should work with their specific device.

Reply:
Very good, then it makes sense to leave as is